Cargo.toml

```diff
@@ -963,6 +966,11 @@ name = "determine_if_only_one_instance_is_running"
 path = "src/determine_if_only_one_instance_is_running.rs"
 
 [[bin]]
+# http://rosettacode.org/wiki/Digital_root
+name = "digital_root"
+path = "src/digital_root.rs"
```

Comment:
Thanks. Looks like that was it. :)

(I haven't actually managed to run the top-level project locally yet, due to unrelated issues with building `openssl-sys-extras` on Windows.)